Yosemite/YosemiteTests/Stores/ReceiptStoreTests.swift

```diff
@@ -474,4 +474,17 @@ private extension ReceiptStoreTests {
                            totalTax: "",
                            attributes: [])
     }
+
+    func makeOrderTaxLine(totalShippingTax: String,
+                          totalTax: String) -> Yosemite.OrderTaxLine {
+        .init(taxID: 123,
+              rateCode: "",
+              rateID: 1,
+              label: "State",
+              isCompoundTaxRate: false,
+              totalTax: totalTax,
+              totalShippingTax: totalShippingTax,
+              ratePercent: 5.5,
+              attributes: [])
```

Comment:
Similar to previous comments - this can be replaced with `.fake()` and `.copy()` for simplicity.

Reply:
Stopped created tax lines in https://github.com/woocommerce/woocommerce-ios/pull/5931/commits/a0d1c2cc8b2d3cea71d2354150cc5ec5712d9ad9